iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/procedure/impl/schema/DeleteDatabaseProcedure.java

```diff
@@ -206,6 +206,7 @@ protected Flow executeFromState(ConfigNodeProcedureEnv env, DeleteStorageGroupSt
             LOG.info(
                 "[DeleteDatabaseProcedure] Database: {} is deleted successfully",
                 deleteDatabaseSchema.getName());
+            env.getConfigManager().getLoadManager().clearPartitionBalancer();
```

Comment:
Whether only the corresponding database table should be clear?

Reply:
My big mistake!!! Fixed now